src/extensions/hive/components/PublishInfo.css

```diff
@@ -0,0 +1,8 @@
+.publishInfo {
```

Comment:
Remove this file and use inline styling in the React component

Reply:
ok

Reply:
moved to inline style